src/views/Dashboard.vue

```diff
@@ -66,7 +66,7 @@
                 <v-btn text small color="primary"
                   id="standalone-addresses-button"
                   class="change-btn"
-                  :disabled="hasBlockerFiling"
+                  :disabled="hasBlockerFiling || coaPending"
```

Comment:
I believe this is redundant. It looks like both properties are set in checkPendingFilings().

Also on line 89.

Reply:
I'll look into this!